std/datetime.d

```diff
@@ -17632,18 +17632,18 @@ assert(Interval!Date(Date(1996, 1, 2), dur!"years"(3)) ==
 
     /++
         The starting point of the interval. It is included in the interval.
-
-        Examples:
---------------------
-assert(Interval!Date(Date(1996, 1, 2), Date(2012, 3, 1)).begin ==
-       Date(1996, 1, 2));
---------------------
       +/
     @property TP begin() const pure nothrow
     {
         return cast(TP)_begin;
     }
 
+    ///
+    unittest
+    {
+        assert(Interval!Date(Date(1996, 1, 2), Date(2012, 3, 1)).begin ==
+            Date(1996, 1, 2));
```

Comment:
If you look at the and of the PR, not all old example follow the same formatting.

I hope I made them all equal